pkg/commands/policy/policy.go

```diff
@@ -90,5 +96,40 @@ func (c *PolicyCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to unmarshal json: %w", err)
 	}
 
-	return nil
+	c.results = results
+
+	return c.Process(ctx)
+}
+
+// Process handles the main logic for handling the results of the policy
+// evaluation.
+func (c *PolicyCommand) Process(ctx context.Context) error {
+	logger := logging.FromContext(ctx)
+
+	var merr error
+	for k, v := range *c.results {
+		logger.DebugContext(ctx, "processing policy decision",
+			"policy_name", k)
+
+		if len(v.MissingApprovals) == 0 {
+			logger.DebugContext(ctx, "no missing approvals for policy",
+				"policy_name", k)
+			continue
+		}
+
+		for _, m := range v.MissingApprovals {
+			c.teams = append(c.teams, m.AssignTeams...)
+			c.users = append(c.users, m.AssignUsers...)
```

Comment:
This seems unnecessary and is only used to try to get the test results you want. I would just test the response message. When you add the functionality to assign reviewer, mock the GitHub client and expect the requests sent contain these reviewers instead (similiar to how plan and apply are tested).

Reply:
Removed from struct and moved all logic (except for parsing flags) into `Process`